lib/Resource.js

```diff
@@ -141,6 +145,14 @@ Resource.prototype.relationships = function(relationName) {
   var self = this;
   var rawRelation = this._raw.relationships[relationName];
 
+  if (!self._base.id) {
+    rawRelation = this._raw.relationships[relationName] = {
```

Comment:
We are already assigning `this._raw.relationships[relationName]` to rawRelation on line 146. Could we not just set relationName to the new object?